packages/docs-data/markdownRenderer.mjs

```diff
@@ -47,4 +47,13 @@ function renderMarkdown(textContent) {
     return marked(textContent, { renderer });
 }
 
-export { renderMarkdown, renderer as markedRenderer };
+const hooks = {
+    // HACKHACK: workaround for https://github.com/palantir/documentalist/issues/248
+    // We are getting inline code snippets wrapped by newlines, which breaks the markdown rendering of multiline JSDoc
+    // comments. We can work around this by removing the newlines. This should work for all practical cases since we
+    // don't expect any JSDoc comment line to simply have a `code` token as its only contents.
+    preprocess: md => md.replace(/\n(\`.*\`)\n/g, "$1"),
```

Comment:
this matched some unrelated markdown, issues visible on the docs home page and other places in the docs:

<img width="828" alt="image" src="https://github.com/palantir/blueprint/assets/723999/916ac059-b038-4476-a044-94a7712fa176">